### PR TITLE
Vagrant config file

### PR DESCRIPTION
**Pull request by malavsky**

### CODE REVIEW

File: shell/nginx.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+              fastcgi_split_path_info ^(.+\.php)(/.*)$;
+              include fastcgi_params;
+              fastcgi_param SCRIPT_FILENAME \$document_root\$fastcgi_script_name;
+              fastcgi_param HTTPS off;
+              fastcgi_read_timeout 600;
+          }
+
+          error_log /var/log/nginx/$1.error.log;
+          access_log /var/log/nginx/$1.access.log;
+      }" >> /etc/nginx/sites-available/symfony
```
Comment (aivus) [inline shell/nginx.sh:23]: How about server name here too?

**Comment by aivus** [comment]
:+1:

File: shell/nginx.sh (hunk 1 above)
Comment (malavsky) [inline shell/nginx.sh:23]: I can to add. But in this there is no sense. In one box we have only one nginx config

Comment (aivus) [inline shell/nginx.sh:23]: ok

**Pull request merged**